lib/src/foundation/bloc.dart

```diff
@@ -110,7 +110,7 @@ class Bloc {
 
     _steps = steps;
     _stepsToIgnore = await _alreadyCompletedSteps;
-    _steps = _steps.where((s) => !_stepsToIgnore.contains(s)).toList();
+    _steps = _steps!=null?_steps.where((s) => !_stepsToIgnore.contains(s)).toList():[];
```

Comment:
What's your problem with spacing? :)

Reply:
I don't know what is the problem, the log error is:

lib/src/foundation/bloc.dart
lib/src/rendering/custom_layout.dart
test/feature_discovery_test.dart
Formatting failed: 1
Error: Process completed with exit code 1.

Reply:
I meant, why not add spaces around the operators?

```dart
_steps = _steps != null ? _steps.where((s) => !_stepsToIgnore.contains(s)).toList() : [];
```

This could probably be rewritten like this:

```dart
_steps = _steps?.where((s) => !_stepsToIgnore.contains(s)).toList() ?? [];
```